src/Storages/RocksDB/StorageSystemRocksDB.cpp

```diff
@@ -101,10 +101,9 @@ void StorageSystemRocksDB::fillData(MutableColumns & res_columns, ContextPtr con
         String database = (*col_database_to_filter)[i].safeGet<const String &>();
         String table = (*col_table_to_filter)[i].safeGet<const String &>();
 
-        auto & rocksdb_table = dynamic_cast<StorageEmbeddedRocksDB &>(*tables[database][table]);
-        auto statistics = rocksdb_table.getRocksDBStatistics();
+        auto statistics = tables[database][table]->getRocksDBStatistics();
         if (!statistics)
-            throw Exception(ErrorCodes::LOGICAL_ERROR, "rocksdb statistics is not enabled");
+            throw Exception(ErrorCodes::LOGICAL_ERROR, "RocksDB statistics is not avaliable");
```

Comment:
Change later not to trigger CI.
```suggestion
            throw Exception(ErrorCodes::LOGICAL_ERROR, "RocksDB statistics is not available");
```

Reply:
`statistics is` -> `statistics are`